server.js

```diff
@@ -39,14 +43,20 @@ app.get("/menu_items", function (req, res) {
   ]);
 });
 
-//Post Method
-app.post("/menu_items", (req, res, next) => {
-  res.send(JSON.stringify(req.body));
+app.post("/food", async (req, res) => {
+  const food = await prisma.food.create({ data: req.body });
+  // console.log(req.body);
+  // const food = "food";
+  res.json(food);
+});
+
+app.delete("/food/:id", (req, res) => {
+  const deletedFoodItem = prisma.food.deleteMany({});
+  res.send(deletedFoodItem);
 });
 
-//Delete by ID Method
-app.delete("/delete/:id", (req, res) => {
-  res.send("Delete by ID API");
+app.patch("/menu_items", (req, res) => {
+  res.send("Api is working");
```

Comment:
I had envisioned a `PUT` here, not a `PATCH`.  The functional difference is that a `PUT` will completely replace the record at ID {whatever} whereas a `PATCH` will modify only a subset of fields.  It's much more difficult to implement a `PATCH` correctly.

Reply:
switched to put